src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdApi.java

```diff
@@ -70,6 +73,43 @@ CaseDetails getCase(String caseRef, String jurisdiction) {
         }
     }
 
+    void attachExceptionRecord(CaseDetails theCase,
+                               Map<String, Object> data,
+                               StartEventResponse event) {
+        String caseRef = String.valueOf(theCase.getId());
+        String jurisdiction = theCase.getJurisdiction();
+        String caseTypeId = theCase.getCaseTypeId();
+        try {
+            CcdAuthenticator authenticator = authenticatorFactory.createForJurisdiction(jurisdiction);
+            attachCall(caseRef, authenticator, data, event.getEventId(), event.getToken(), jurisdiction, caseTypeId);
+        } catch (FeignException e) {
+            throw error(e, "Internal Error: submitting attach file event failed case: %s Error: %s",
+                caseRef, e.status());
+        }
+    }
+
+    private void attachCall(String caseRef,
+                            CcdAuthenticator authenticator,
+                            Map<String, Object> data,
+                            String eventId,
+                            String token, String jurisdiction,
+                            String caseTypeId) {
+        feignCcdApi.submitEventForCaseWorker(
+            authenticator.getUserToken(),
+            authenticator.getServiceToken(),
+            authenticator.getUserDetails().getId(),
+            jurisdiction,
+            caseTypeId,
+            caseRef,
+            true,
+            CaseDataContent.builder()
+                .data(data)
+                .event(Event.builder().id(eventId).build())
+                .eventToken(token)
+                .build()
```

Comment:
i think we should provide a summary. even if it something like just "Attaching document to original case"

Reply:
not entirely sure what you mean ...
a log statement ?

Reply:
Unless I'm mixing up APIs. I think I saw `.summary` call on one of those ccd builders. Just saying. If it's not there then there's nothing to worry about. Well, there's nothing to worry about anyway actually

Reply:
yeah there is a description and summary fields ... 
no requirements on what should go into those though.
should I fill those out ?

Reply:
Just a human-alike sentence. Consider it as a brief descriptive comment on what has happened

Reply:
ok ... ill add something.

Reply:
Done ... nice descriptive summary :smile:
I also added asserts that check the information about token id and summary
(almost made a mistake with parameters being strings got them mixed around until my test told me otherwise ;) )